mm/2s2h/Enhancements/Cycle/EndOfCycle.cpp

```diff
@@ -7,10 +7,13 @@ extern "C" {
 }
 
 SaveInfo saveInfoCopy;
+s32 timeSpeedOffsetCopy;
 
 void RegisterEndOfCycleSaveHooks() {
-    GameInteractor::Instance->RegisterGameHook<GameInteractor::BeforeEndOfCycleSave>(
-        []() { memcpy(&saveInfoCopy, &gSaveContext.save.saveInfo, sizeof(SaveInfo)); });
+    GameInteractor::Instance->RegisterGameHook<GameInteractor::BeforeEndOfCycleSave>([]() {
+        memcpy(&saveInfoCopy, &gSaveContext.save.saveInfo, sizeof(SaveInfo));
+        memcpy(&timeSpeedOffsetCopy, &gSaveContext.save.timeSpeedOffset, sizeof(s32));
```

Comment:
Doing a `memcpy` for a primitive value like an integer is overkill. You can just do a `var = value` assignment instead.
```suggestion
        timeSpeedOffsetCopy = gSaveContext.save.timeSpeedOffset;
```